### ISSUE TITLE
cant build PHP 8.1.1 with mcrypt 1.0.4

### ISSUE THREAD

**Issue by cyrax13** [php/pecl-encryption-mcrypt#8]
Hello there)

Simply try to build docker image for PHP 8.1.1 using alpine linux, source code from here: [https://github.com/docker-library/php/blob/c3ee24b4280eff1b55b7a22e527532617e56bcf0/8.1/alpine3.15/fpm/Dockerfile](https://github.com/docker-library/php/blob/c3ee24b4280eff1b55b7a22e527532617e56bcf0/8.1/alpine3.15/fpm/Dockerfile)

In my Dockerfile:
```
RUN pecl install mcrypt
```

Results - build failed:
```
WARNING: channel "pecl.php.net" has updated its protocols, use "pecl channel-update pecl.php.net" to update
pecl/mcrypt requires PHP (version >= 7.2.0, version <= 8.1.0, excluded versions: 8.1.0), installed version is 8.1.1
No valid packages found
install failed
```

Maybe it is worth increasing the version range in the file [package.xml](https://github.com/php/pecl-encryption-mcrypt/blob/master/package.xml)
```
<max>8.1.0</max>
<exclude>8.1.0</exclude>
```
something like this - 8.1.99?

Can you fix  this issue? Thanks)

**Comment by Zaryaab-Shahbaz** [comment]
Facing the same issue.

`RUN pecl install mcrypt-1.0.4 `

PHP Image
`FROM php:8.1.1-fpm-alpine`

Any help would be appreciated.

**Comment by theLine** [comment]
Have a look into the comment I’ve posted in https://github.com/php/pecl-encryption-mcrypt/issues/7#issuecomment-996524649 for a workaround

**Comment by cyrax13** [comment]
@theLine greetings. Thanks for the comment, but the way indicated in it does not help(

PS: PHP 8.1.2 released, same problem...

**Comment by theLine** [comment]
@cyrax13 Can you share your `Dockerfile` or at least the error you're getting?

I could build `mcrypt-1.0.4` from source with the commands I've provided in the linked issue earlier today without any issues.
FYI: I used `php:8.1-cli` and `php:8.1-fpm` as base images.

**Comment by technicalguru** [comment]
Same issue with building latest PHP 8.1.x version. The 8.1.x versions are excluded by the package.xml dependency. Any chance to get this fixed soon?
```
File //mcrypt-1.0.4.tgz downloaded
pecl/mcrypt requires PHP (version >= 7.2.0, version <= 8.1.0, excluded versions: 8.1.0), installed version is 8.1.3
No valid packages found
install failed
```

**Comment by zeriyoshi** [comment]
fixed #9 
For now, 1.0.4 can be used successfully if PECL is not used.

**Comment by technicalguru** [comment]
Thanks. I was able to build it. Waiting for 1.0.5 release then :)

**Comment by zeriyoshi** [comment]
This issue has been fixed in 1.0.5.

**Comment by cyrax13** [comment]
Thanks for the fix, closed)
